AWSAppSyncClient.xcodeproj/project.pbxproj

```diff
@@ -2049,7 +2049,6 @@
 				B48056BC22A992E200E4F742 /* AppSyncSubscriptionWithSyncTests.swift in Sources */,
 				FAC28D3B221B0F09000F84CD /* CacheKeyTests.swift in Sources */,
 				2171809123FDB28100E520C9 /* UserPoolsBasedConnectionPoolTests.swift in Sources */,
-				B47DECF6235C027F003A8127 /* APIKeyAuthInterceptorTests.swift in Sources */,
```

Comment:
Why was this file removed?

Reply:
It wasn't removed with this commit; it must have been Xcode removing stray file references.